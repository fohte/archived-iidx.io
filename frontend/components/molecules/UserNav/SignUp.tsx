import { Button } from 'antd'
import Link from 'next/link'

const SignUp: React.SFC = () => (
  <Link href="/signup" prefetch>
    <Button>SignUp</Button>
  </Link>
)

export default SignUp
