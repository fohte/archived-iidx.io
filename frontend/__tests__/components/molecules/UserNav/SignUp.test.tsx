import * as React from 'react'
import * as renderer from 'react-test-renderer'

import SignUp from '@app/components/molecules/UserNav/SignUp'

describe('SignUp', () => {
  it('renders correctly', () => {
    const component = renderer.create(<SignUp />)
    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
