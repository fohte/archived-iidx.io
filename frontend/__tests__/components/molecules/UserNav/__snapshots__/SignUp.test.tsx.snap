// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp renders correctly 1`] = `
<button
  className="ant-btn"
  onClick={[Function]}
  type="button"
>
  <span>
    SignUp
  </span>
</button>
`;
